example/src/main/java/com/transferzero/test/Application.java

```diff
@@ -7,12 +7,15 @@
 import com.transferzero.sdk.api.SendersApi;
 import com.transferzero.sdk.api.TransactionsApi;
 import com.transferzero.sdk.model.*;
-import java.time.LocalDate;
+import okhttp3.Request;
+import okhttp3.Response;
 
 import java.math.BigDecimal;
+import java.time.LocalDate;
 import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.UUID;
-import java.util.*;
```

Comment:
`util.*` should be sufficient here, not sure why it's removed? Is it because your IDE wasn't picking up the imports?